rdkit/TestRunner.py

```diff
@@ -50,8 +63,10 @@ def RunScript(script, doLongTests, verbose):
     # setting environment allows this setting to recursively pass to all child
     # processes
     os.environ['PYTHON_TEST_FAILFAST'] = '1'
-  if len(sys.argv) == 3 and sys.argv[1] == '--testDir':
+  if len(sys.argv) >= 3 and sys.argv[1] == '--testDir':
     os.chdir(sys.argv[2])
+  if len(sys.argv) >= 5 and sys.argv[3] == '--buildType':
```

Comment:
Does this need to be done with an environment variable? Wouldn't it be simpler to just do it with a global?

Reply:
I think a global would not work, since `RunTest()` uses `subprocess.call()` to run the tests in a separate process.

Reply:
Anyway, the env var will only exist in the python process, it will not be exported to the shell running the tests.